app/jobs/generate_interstream_delta_job.rb

```diff
@@ -100,7 +100,7 @@ def perform(stream)
 
     # Attach Files
     unless current_stream_dump.interstream_delta
-      current_stream_dump.interstream_delta = InterstreamDelta.create(normalized_dump: current_stream_dump, stream: stream)
+      current_stream_dump.interstream_delta = InterstreamDelta.create(normalized_dump: current_stream_dump)
```

Comment:
Can't ActiveRecord do this for us?

```suggestion
  current_stream_dump.create_interstream_delta
```

(untested.)

Reply:
Seems to work. Thanks!